### PR TITLE
reorganize and update readme

### LINKED ISSUE #213: new improved README, move docs to wiki
to help people understand what marky is trying to do, i'd like to rework the readme and move much of the information to the wiki as docs. thoughts @revin?

### PR DESCRIPTION
**Pull request by ashleygwilliams**
fixes #213 

the README was too focused on how marky works, not how to use marky. this PR reorders the sections to privilege users of marky over developers of marky. also it gives an intro which better summarizes what marky is, and is trying to be.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,24 +6,25 @@
...
-- Parses and sanitizes `package.description` as markdown.
-- Applies CSS classes to redundant content that closely matches npm package name and description.
-- Applies CSS classes to badge images, so we can do something interesting with them one day.
+`marky-markdown` is a markdown parser, written in NodeJS, that aims for 
+parity with [GitHub-style markdown]. It is built on top of [`markdown-it`],
+a [CommonMark] markdown parser. You can use marky-markdown: 
+
+- [programmatically in NodeJS]
+- [in your terminal]
+- [in the browser] (does not yet support syntax highlighting)
```
Comment (revin) [inline README.md]: huh, looks like GH is interpreting this as a full link with the parenthetical part as the link destination

Comment (ashleygwilliams) [inline README.md]: lol, lemme fix that

**Comment by revin** [comment]
This is a ✨ fantastic ✨  idea, thanks for thinking of it.

Are you done tweaking it? I'll merge when you're ready, or you go ahead or whatever (you're the boss, after all 😛)

**Comment by revin** [comment]
oh wait you removed the WIP title, never mind!

**Pull request merged**